docs/adr/0016-pr-review.md

```diff
@@ -0,0 +1,73 @@
+# Reviewing Pull Requests
+
+* Status: accepted
+* Deciders: tumido, HumairAK, lars, goern, 4n4nd
+* Date: 2021-05-20
+
+Technical Story: [#93](https://github.com/operate-first/blueprint/issues/93)
+
+## Context and Problem Statement
+
+Each community has its own `contributing.md` file which outlines how are users expected to contribute to the community. That guide for contributions should also clearly state what will happen once a PR is submitted and how reviews are performed.
+
+
+## Considered Options
+
+* Using inherited processes from AI-CoE - Prow
+* Building upon Prow and modifying and evolving them to operations needs
+
+## Decision Outcome
+
+Chosen option: "Building upon Prow and modifying and evolving them to operations needs"
+
+### Prerequisites
+
+We use Prow to automate review process. That means we use following setup:
+
+1. Each repository contains an `OWNERS` file:
+   1. There is always one `OWNERS` file in a repository root folder
+   2. If needed, additional `OWNERS` files can be created within nested folders. This file takes precedence for all resources within this folder.
+2. `OWNERS` file contains several entries:
+   1. `approvers` is a list of GitHub users that are allowed to use `/approve` chat-ops command. Role of approvers is to gate PRs - they verify that changes in PRs respect our architectural decisions and are in line with the heading of the project. See [upstream docs][4] for better understanding of this role.
+   2. `reviewers` is a list of GitHub users that are allowed to use `/lgtm` chat-ops command. Their role is to ensure changes in PRs are implemented properly and meet our coding standards. See [upstream docs][4] for details.
+
+### Process
+
+1. A PR is submited
+2. Prow (represented by a Sesheta bot-account) will pick a selection of reviewers from corresponding `OWNERS` file (based on modified files in the PR).
+3. Reviewers assess the changes from programming/coding point of view and either provide feedback or respond with `/lgtm` chat-ops command.
+   1. Once a `/lgtm` command is sent, Prow (Sesheta) will add a `lgtm` label to the PR. This label will get removed if there are any new pushes to the PR.
+   2. Additionally Prow will ask users to assign a specific `approver` to the PR if there's no assignment yet.
+4. Approvers evaluate architectural aspects of the PR and either provide feedback or respond with `/approve` chat-ops command.
+   1. Once an `/approve` command is sent, Prow (Sesheta) will add an `approve` label to the PR. This label stays after consequent pushes to the PR.
```

Comment:
"Consequent" implies "caused by the approval", whereas "subsequent" just means "following". Which one do you mean here?

Reply:
I've meant "subsequent", thank you :slightly_smiling_face: